tests/test_utils/graphs.py

```diff
@@ -210,6 +215,10 @@ def example_graph_random(
     node_ids = range(n_nodes)
     if feature_size is None:
         nodes = pd.DataFrame(index=node_ids)
+    elif random_features:
+        nodes = pd.DataFrame(
+            np.random.random((len(node_ids), feature_size)), index=node_ids
```

Comment:
Just to understand, did you try my alternate suggestion of swapping the `ones` for `random` to avoid having to add yet another parameter?

Reply:
No I didn't sorry, tried it out and it works find 👍